static/js/services/report-view-model-generator.js

```diff
@@ -0,0 +1,65 @@
+var _ = require('underscore');
+
+angular.module('inboxServices').factory('ReportViewModelGenerator',
+  function(
+    DB,
+    FormatDataRecord,
+    LineageModelGenerator
+  ) {
+    'ngInject';
+    'use strict';
+
+    var getFields = function(results, values, labelPrefix, depth) {
+      if (depth > 3) {
```

Comment:
Can you add a comment explaining what `depth` is? This maxing out at 3 looks weird